layouts/Workspaces/components/WorkspacesSidebar.tsx

```diff
@@ -13,7 +13,7 @@ import Link from 'next/link';
 import { useRouter } from 'next/router';
 import { useUserQuery } from '@/graphql/user/UserQuery.generated';
 import { SkeletonList } from '@/components/SkeletonList';
-import AddWorkspaceModal from '@/layouts/components/AddWorkspaceModal';
+import AddWorkspaceModal from '@/layouts/Workspaces/components/AddWorkspaceModal';
```

Comment:
I moved non page components into our `components` directory. We shouldn't have them in `/pages` because next treats everything in that directory as a page, therefore when building it will build all components as pages and drastically increase build times.